routing_optimization_over_server/README.md

```diff
@@ -0,0 +1,15 @@
+# cuOpt Server Notebooks
+
+Contains a collection of Jupyter Notebooks that outline how cuOpt self hosted service can be used to solve a wide variety of problems.
+
+To use these notebooks, you need to install [cuopt-sh-client](https://registry.ngc.nvidia.com/orgs/j9mrpofbmtxd/teams/cuopt_self_hosted/resources/cuopt-self-hosted-client) available in NGC in Resources tab.
```

Comment:
The self hosted client will be available in pypi, so lets add it to requirements.txt.